SECURITY.md

```diff
@@ -34,6 +34,7 @@ Please read prior audit reports for projects that use Vyper here:
 | Project | Version | Report Link |
 | ------- | ------- | ----------- |
 | [Uniswap](https://uniswap.io) | 35038d2 | https://medium.com/consensys-diligence/uniswap-audit-b90335ac007 |
+| [Computable](https://www.computable.io/) | 0.1.0b10 | https://github.com/trailofbits/publications/blob/master/reviews/computable.pdf |
```

Comment:
maybe link to the raw file so people don't have to use the github reader

https://github.com/trailofbits/publications/raw/master/reviews/computable.pdf